.travis.yml

```diff
@@ -0,0 +1,89 @@
+language: bash
+
+sudo: required
+
+services:
+  - docker
+
+addons:
+  apt:
+    packages:
+      - docker-ce
+      - tree
+
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
+notifications:
+  email:
+    on_success: never
+    on_failure: always
+
+env:
+  global:
+    - DOCKER_COMPOSE_VERSION=1.24.0
+    - THREADS=4
+    - COMPOSER_GLOBAL_OPTIONS="--no-interaction --no-suggest -o"
+    # Disable xDebug, because it is not required for testing.
+    - PHP_XDEBUG=0
+  matrix:
+    - CHECK_CREATED_DEV_ENV=true
+    - CHECK_CREATED_DEV_ENV=true DEPENDENCIES="--prefer-lowest"
+    - CHECK_PHPUNIT_SUPPORT=true
+    - CHECK_BEHAT_SUPPORT=true
+
+matrix:
+  fast_finish: true
+
+before_install:
+  - |
+    sudo rm /usr/local/bin/docker-compose
+    curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+    chmod +x docker-compose && sudo mv docker-compose /usr/local/bin
+  # Create a Drupal module from the template. (Composer 1.7 latest is being used because 1.8.x has a performance issue.)
+  # TL;DR You can not just pass --repository='{"type": "path", "url": "/app"}' (a JSON object) as a parameter thanks
+  # for the YML syntax o.O
+  - docker run --rm -it -v $TRAVIS_BUILD_DIR:/app -v /tmp:/tmp --user $(id -u):$(id -g) -e COMPOSER_MIRROR_PATH_REPOS=1 composer:1.7 create-project --repository=$(docker run --rm -it composer:1.7 php -r "echo json_encode(['type' => 'path', 'url' => '/app']);") -s dev pronovix/drupal-module-template /tmp/my_module
+  # Dummy permission fix for Travis because the default user's id is not 1000 (which is also the id of the "wodby" user
+  # in the containers).
+  # uid=2000(travis) gid=2000(travis) groups=2000(travis),999(docker)
+  - cd /tmp/my_module && sudo chown -R 1000:travis . $HOME/.composer && tree -a
```

Comment:
See my comment in the other travis.yml file.

Reply:
So can we make this comment also a little bit clearer for newcomers like me?

Reply:
I tried to did that by adding the output of `id` as a comment :D

Reply:
I mean this should be enough for now, because this is (hopefully) a one time only tweak.
`
  # Dummy permission fix for Travis because the default user's id is not 1000 (which is also the id of the "wodby" user
  # in the containers).
  # uid=2000(travis) gid=2000(travis) groups=2000(travis),999(docker)
`

Reply:
Nevermind, updated to comment, please check again. (I wanted to keep this short :) ).